pmm/pmm2-ui-tests.groovy

```diff
@@ -226,11 +230,11 @@ pipeline {
                         waitForContainer('pmm-agent_mysql_5_7', "Server hostname (bind-address):")
                         waitForContainer('pmm-agent_postgres', 'PostgreSQL init process complete; ready for start up.')
                         sh """
+                            docker exec pmm-server change-admin-password \${ADMIN_PASSWORD}
```

Comment:
Do we need to change it if the user hasn't changed the default `admin`? In that case maybe not...

Reply:
We are changing it by default for UI tests job. It takes a job parameter which has default value